api.lua

```diff
@@ -1343,14 +1344,14 @@ function get_sign_formspec(pos, nodename)
 		"image[0.1,2.4;7,1;signs_lib_sign_color_palette.png]",
 		"textarea[0.15,-0.2;6.3,2.8;text;;" .. minetest.formspec_escape(txt) .. "]",
 		"button_exit[3.7,3.4;2,1;ok;" .. S("Write") .. "]",
-		"label[0.3,3.4;Unicode font]",
+		"label[0.3,3.4;"..FS("Unicode font").."]",
 		"image_button[0.6,3.7;1,0.6;signs_lib_switch_" .. state .. ".png;uni_"
 			.. state .. ";;;false;signs_lib_switch_interm.png]",
 	}
 
 	if minetest.registered_nodes[nodename].allow_widefont then
 		state = meta:get_int("widefont") == 1 and "on" or "off"
-		formspec[#formspec+1] = "label[2.1,3.4;Wide font]"
+		formspec[#formspec+1] = "label[2.1,3.4;"..FS("Wide font").."]"
```

Comment:
Why do these need to be escaped? I thought escaping was only needed for user-editable values.

Reply:
translated strings are almost user-editable. But you may be right that translator already escapes them?